playbooks/ooi.yaml

```diff
@@ -0,0 +1,146 @@
+---
+- hosts: packstack
+  become: yes
+  become_method: sudo
+  tasks:
+    # XXX Hardcoded for OCCI 1.1
+    - name: Configure OOI repository
+      yum_repository:
+        name: ooi-centos-7-x86_6
+        description: Repository for OOI
+        baseurl: http://repository.egi.eu/community/software/ooi/occi-1.1/releases/centos/7/x86_64/RPMS/
+        gpgcheck: no
+        gpgkey: http://repository.egi.eu/community/keys/APPDBCOMM-DEB-PGP-KEY.asc
+
+    - name: Install OOI
```

Comment:
maintain a common syntax for tasks. instead of one-liner (var=value) you should have one value per line : (var: value)
So instead of ` yum: name=python-ooi state=installed`, 

```
yum:
  name: python-ooi
  state: instaled
```
This format also makes it easier to catch what has changed when you do a diff (changes are only one per line)

Reply:
Done.